packages/fork-choice/src/forkChoice/store.ts

```diff
@@ -22,6 +22,7 @@ export type CheckpointWithHex = phase0.Checkpoint & {rootHex: RootHex};
  */
 export interface IForkChoiceStore {
   currentSlot: Slot;
+  secondsIntoSlot: number;
```

Comment:
I'm not sure if we need this in store, I don't see this in the spec as well. Maybe if we always have `blockReceptionTime`, we don't need this anymore? this does not look like other variables which hold the state of the forkchoice.

Reply:
This is actually needed to make the spec tests work as there is a step in it to compare `time`: over here https://github.com/ChainSafe/lodestar/pull/3540/files#diff-49a2a7fb7007eb44eff9a66634a30ff5475e4529132d81b6960675a86b98e1b3R108-R109

Reply:
Lighthouse doesn't include this data point in the store and can pass spec tests https://github.com/sigp/lighthouse/blob/unstable/consensus/fork_choice/src/fork_choice_store.rs

Reply:
because they seem to be matching only slot on the check_time,  which was fine before proposer boost as there were time ticks in tests only on slot boundaries, but now we get time ticks even within the slot
https://github.com/sigp/lighthouse/blob/unstable/testing/ef_tests/src/cases/fork_choice.rs#L436:L442